tests/test_diag_data_dictionary_spec.py

```diff
@@ -264,6 +264,7 @@ def test_initialization(self) -> None:
             short_name="mux_case1_struct",
             long_name=None,
             description=None,
+            is_visible_raw=None,
```

Comment:
Didn't you remove `is_visible_raw`  in a previous PR?

Reply:
yes, #240 (I suppose). seems like I made a mistake back then. (IS-VISIBLE is only part of STRUCTURE, but not BASIC-STRUCTURE...)